models/uniswap/avalanche_c/uniswap_avalanche_c_trades.sql

```diff
@@ -0,0 +1,42 @@
+{{ config(
+        alias = 'trades'
```

Comment:
We may also need the schema here:
```suggestion
        schema = 'uniswap_avalanche_c',
        alias = 'trades'
```